resource/systemd/unit/preparer.go

```diff
@@ -0,0 +1,129 @@
+// Copyright © 2016 Asteris, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package unit
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/asteris-llc/converge/load/registry"
+	"github.com/asteris-llc/converge/resource"
+	"github.com/asteris-llc/converge/resource/systemd"
+)
+
+// Preparer for Systemd Unit
+type Preparer struct {
+	// The name of this unit as in "foo.service"
+	Name string `hcl:"name"`
+
+	// Active determines whether this unit should be active or inactive
+	Active string `hcl:"active"`
+
+	/* State is the UnitFileState of the unit.
```

Comment:
Could you add some whitespace lines in this comment? It's nearly unreadable as-is.